addons/account/models/account_report.py

```diff
@@ -691,6 +691,11 @@ def write(self, vals):
 
         return result
 
+    def copy_data(self, default=None):
+        vals_list = super().copy_data(default=default)
+        return [dict(vals, label=self.env._("%s (copy)", expression.label)) for expression, vals in zip(self, vals_list)]
```

Comment:
Well it's a very standard function used on a lot of models that enable the copy. Right now when trying to duplicate an expression in the interface (I know it might be dump to do it but I tried when doing tests for the trial balance), it just raise a validation error because the name is already taken. But no it the name already contains "copy", it will just add another "copy". I can remove this branch if you think it's useless